src/features/events/hooks/useEventLocation.ts

```diff
@@ -0,0 +1,21 @@
+import { loadListIfNecessary } from 'core/caching/cacheUtils';
+import { ZetkinLocation } from 'utils/types/zetkin';
+import { locationsLoad, locationsLoaded } from '../store';
+import { useApiClient, useAppDispatch, useAppSelector } from 'core/hooks';
+
+export default function useEventLocation(
```

Comment:
The singular name of this hook implies to me that it returns a singular thing, but it returns a list. Should it be called `useEventLocations` instead?